tests/e2e/e2e_test.go

```diff
@@ -88,6 +92,82 @@ func (s *IntegrationTestSuite) TestIBC() {
 	s.testIBCTokenTransfer()
 	s.testMultihopIBCTokenTransfer()
 	s.testFailedMultihopIBCTokenTransfer()
+
+	// stop hermes0 to prevent hermes0 relaying transactions
+	s.Require().NoError(s.dkrPool.Purge(s.hermesResource0))
+	HermesResource0Purged = true
+	s.testIBCBypassMsg()
+}
+
+func (s *IntegrationTestSuite) testIBCBypassMsg() {
+
+	// submit gov proposal to change bypass-msg param to
+	// ["/ibc.core.channel.v1.MsgRecvPacket",
+	//  "/ibc.core.channel.v1.MsgAcknowledgement",
+	//  "/ibc.core.client.v1.MsgUpdateClient"]
+	submitterAddr := s.chainA.validators[0].keyInfo.GetAddress()
+	submitter := submitterAddr.String()
+	proposalCounter++
+	s.govProposeNewBypassMsgs([]string{
+		sdk.MsgTypeURL(&ibcchanneltypes.MsgRecvPacket{}),
+		sdk.MsgTypeURL(&ibcchanneltypes.MsgAcknowledgement{}),
+		sdk.MsgTypeURL(&ibcclienttypes.MsgUpdateClient{})}, proposalCounter, submitter, standardFees.String())
+
+	// use hermes1 to test default ibc bypass-msg
```

Comment:
Would it make sense to break these three tests into their own functions?

Reply:
we might only need one hermes which is this one in this PR. 
- before, we have one hermes running and doing ibc transfer and PFM test.
- now, i need to test hermes with two configs, one hermes' config is configured to pay fees, one not. So that I can do transactions async (with hermes not running), and i can choose which config i use to "clear" the submitted transactions. 

i will add a comment in the code, and may open an issue to refactor two instances hermes into one, to do all test without hermes running.

Reply:
we might only need one hermes which is this one in this PR. 
- before, we have one hermes running and doing ibc transfer and PFM test.
- now, for bypass-min-fee test, i want to control if hermes paying fees or not, so there are two hermes configs, one hermes' config is configured to pay fees, one not. So that I can do transactions async (with hermes not running), and i can choose which config i use to "clear" the submitted transactions. 

i will add a comment in the code, and may open an issue to refactor two instances hermes into one, to do all test without hermes running.